nf_core/pipeline-template/.github/workflows/linting.yml

```
@@ -58,6 +58,15 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GITHUB_PR_COMMIT: ${{ github.event.pull_request.head.sha }}
         run: nf-core -l lint_log.txt pipelines lint --dir ${GITHUB_WORKSPACE} --markdown lint_results.md
+        if: ${{ github.base_ref == 'dev' }}
```

Hello,

This suggestion is to avoid a regression whereby PRs to other feature branches would stop running `nf-core lint`

```suggestion
        if: ${{ github.base_ref != 'master' }}
```

---

Agreed 👍🏻